Tasks/AzureCloudPowerShellDeploymentV1/Utility.ps1

```diff
@@ -134,6 +148,7 @@ function Get-DiagnosticsExtensions($storageAccount, $extensionsPath, $storageAcc
                             try
                             {
                                 $publicConfigStorageKey = Get-AzureStoragePrimaryKey $publicConfigStorageAccountName $useArmStorage.IsPresent
+                               
```

Comment:
Why add an empty line?

Reply:
This empty line existed before code changes.

Reply:
Removed it now and updated the PR. Thanks